### ISSUE TITLE
Error if there is an array index in a case statement

### ISSUE THREAD

**Issue by gustason** [BalticAmadeus/AblFormatter#222]
For example,
``` abl
case myArray[i]:
    when "index" then
        ttPerson.personIndex = integer(cText).
end case.
```

gets parsed like this:
![image](https://github.com/user-attachments/assets/20a45644-9914-45e0-b9ec-f4e9a899ff7d)

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/92

**Comment by eglekaz** [comment]
Fixed with version 0.0.43 of tree-sitter-abl
